improver_tests/ensemble_copula_coupling/test_utilities.py

```diff
@@ -331,19 +331,22 @@ class Test_restore_non_percentile_dimensions(IrisTest):
 
     def setUp(self):
         """Set up temperature cube."""
-        self.current_temperature_forecast_cube = set_up_percentile_cube(
+        self.cube = set_up_percentile_cube(
             np.sort(ECC_TEMPERATURE_REALIZATIONS, axis=0),
             np.array([10, 50, 90], dtype=np.float32),
         )
+        # function is designed to reshape an input data array with dimensions of
+        # "percentiles x points" - generate suitable input data
+        points_data = [self.cube.data[i].flatten() for i in range(3)]
+        self.input_data = np.array(points_data)
 
     def test_basic(self):
         """
         Basic test that the result is a numpy array with the expected contents.
         """
-        cube = self.current_temperature_forecast_cube
-        plen = len(cube.coord("percentile").points)
+        plen = len(self.cube.coord("percentile").points)
```

Comment:
The doc-string suggests this ought to be checking values as well as type. Instead you could just add a check to the test below that an array is returned and delete this one. The two tests are otherwise identical.